src/main/java/pl/winiarczyk/lesson4/homework/JavaCollections1_2/MyList.java

```diff
@@ -0,0 +1,61 @@
+package pl.winiarczyk.lesson4.homework.JavaCollections1_2;
+
+public class MyList implements OwnList {
+
+    int[] ownList = {1, 2, 3, 4, 5};
+
+    @Override
+    public int size() {
+        return ownList.length;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        boolean isEmpty = false;
+        if (ownList.length == 0) {
+            isEmpty = true;
+            return isEmpty;
+        } else {
+            isEmpty = false;
+            return false;
+        }
+    }
+
+    @Override
+    public void add(Integer element) {
+        int n = 0;
+        int newList[] = new int[n + 1];
+        for (int i = 0; i < n; i++) {
+            newList[i] = ownList[i];
+            newList[n] = element;
```

Comment:
Czy n zawsze nie będzie tutaj 0? 
`new int[n + 1];`
więc to n+1 zawsze będzie 1?